app/src/ui/release-notes/release-notes-dialog.tsx

```diff
@@ -22,8 +23,8 @@ const ReleaseNoteHeaderRightUri = encodePathAsUrl(
   'static/release-note-header-right.svg'
 )
 
-const externalContributionRe = /^(.*)(#\d+)(.*)(@[a-zA-Z0-9\-]+)!(.*)$/
-const otherContributionRe = /^(.*)(#\d+)(.*)$/
+const externalContributionRe = /^(.*?)((\s?\#\d+\s?)+)(.*)(@[a-zA-Z0-9\-]+)!(.*)$/
```

Comment:
This regex is getting pretty gnarly - are we able to write some tests against this work to sanity check us against future regressions?

Reply:
Sure thing